external/openglcts/modules/common/glcNearestEdgeTests.cpp

```diff
@@ -285,8 +285,9 @@ void NearestEdgeTestCase::renderQuad ()
 
 	// Apply offset of almost half a texel to the texture coordinates.
 	DE_ASSERT(m_offsetSign == 1.0f || m_offsetSign == -1.0f);
-	const float offsetWidth		= 0.499f / static_cast<float>(m_width);
-	const float offsetHeight	= 0.499f / static_cast<float>(m_height);
+	const float offset			= 0.5f - powf(2.0f, -8.0f);
```

Comment:
doesn't build on some toolchains due to powf(). See the Travis run

Reply:
I suppose I can just use pow here due to C++, I'm a bit C biased ;)